Resources/Private/Styles/scss/Includes/_Element.scss

```diff
@@ -1,18 +1,16 @@
-#{select("Neos.Form.Builder:ElementCollection")} {
-  margin-top: 30px;
+[data-_neos-form-builder-type="Neos.Form.Builder:ElementCollection"] {
+    margin-top: 30px;
+    & > [data-_neos-form-builder-type] {
+        position: relative;
```

Comment:
The default form template and the fusion template have a wrapping `div.clearfix` which is between the `ElementCollection` and the element with the attribute. This causes this selector to not apply.